presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -856,6 +858,30 @@ public void commitInsert(ConnectorSession session, ConnectorInsertTableHandle in
             for (CompletableFuture<?> fileRenameFuture : fileRenameFutures) {
                 MoreFutures.getFutureValue(fileRenameFuture, PrestoException.class);
             }
+
+            if (handle.getWritePath().isPresent()) {
```

Comment:
I'm pretty sure I just merged something that conflicts with this logically. The new way of writing `handle.getWritePath` is `locationService.writePathRoot(handle.getLocationHandle())`